### DISCUSSION TITLE
Issue Tracker: Current Development

### DISCUSSION THREAD

**Discussion by nsmela** [qatrackplus/qatrackplus#597] [General]
Hey there,

What's the current status for Issue Tracker?  I see it can be enabled, but we had issues with it (couldn't create issues, error 500 I think). Is it a feature still in the works, or is it being phased out?

**Comment by randlet** [comment]
It is unsupported with no current plans of it being developed further.  @Ryan
Bottema ***@***.***> developed it as an internal tool I think.
